consensus/babe/babe.go

```diff
@@ -43,7 +43,7 @@ type Session struct {
 	authorityData  []AuthorityData
 	epochThreshold *big.Int // validator threshold for this epoch
 	txQueue        *tx.PriorityQueue
-	slotToProof    map[uint64][]byte  // for slots where we are a producer, store the vrf output+proof
+	slotToProof    map[uint64][]byte  // for slots where we are a producer, store the vrf output (bytes 0-32) + proof (bytes 32-96)
```

Comment:
I think we could make our lives easier by using a struct instead of bytes. Even just using an alias for `[]byte` and defining `GetProof()`, `GetOutput()` methods would be great.

Reply:
If scale is the issue here, perhaps we can open an issue and update this once Ed refactors the codec.

Reply:
Good point, I commented something similar on on the #522

Reply:
okay i'll update it